src/Makefile.am

```diff
@@ -99,6 +99,7 @@ BITCOIN_CORE_H = \
   compressor.h \
   consensus/consensus.h \
   consensus/params.h \
+  consensus/softforks.h \
```

Comment:
If we're going to eventually use this for Hardfork consensus changes too, maybe it would be better to have a more generic name for the file/module: forks, changes, upgrades?